nautobot_design_builder/models.py

```diff
@@ -443,6 +488,15 @@ def __sub__(self, other: "ChangeSet"):
             .values_list("_design_object_id", flat=True)
         )
 
+    # TODO: check if this method is it's necessary
```

Comment:
If we don't have a unit test showing this is necessary then we either need to write a unit test showing its necessity or we need to removed the unneeded code.

Reply:
It is tested here: https://github.com/nautobot/nautobot-app-design-builder/pull/195/files/3cbce7b44505b0b6f0d18a97fb3f1a3e8e180e0a#diff-d8ee4baf558f4cc575ebf16c207ab6d196397fa5f3bb0acf68c61328768a43d9R343
The comment is a leftover